src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,22 @@
+package seedu.addressbook.commands;
+
+
+/**
+ * Sorts all people in the address book by name in ascending order.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ":\n" 
+            + "Sort all persons in the address book in ascending order.\n"
+            + "Example: " + COMMAND_WORD;
+
+    public static final String MESSAGE_SUCCESS = "Address book has been sorted by ascending order!";
+
```

Comment:
Very very small nitpick. Ascending order does not have meaning when referring to names. If you want to be accurate, it should be lexicographical order or something.